app/layout.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import React from 'react';
+import { Layout } from '../components/Layout';
+import '../styles/dashboard.css';
+import '../styles/globals.css';
+const queryClient = new QueryClient();
+
+export default function RootLayout({
```

Comment:
We should avoid default exports in favor of explicit exports
```suggestion
export function RootLayout({
```